spec/rspec/expectations/configuration_spec.rb

```diff
@@ -47,7 +45,13 @@ class << rspec_dup; undef configuration; end
       end
 
       context 'on an interpreter that does not provide BasicObject', :uses_should, :unless => defined?(::BasicObject) do
-        before { RSpec::Expectations::Syntax.disable_should(Delegator) }
+        around do |example|
+          in_sub_process_if_possible do
```

Comment:
Good point, I actually was meaning to circle back to this due to issues elsewhere meaning it all pended!